Support_Your_Locals/Startup.cs

```diff
@@ -34,7 +34,6 @@ public void ConfigureServices(IServiceCollection services)
                 InitialCatalog = Configuration["Database:Name"],
                 UserID = Configuration["Database:Username"],
                 Password = Configuration["Database:Password"],
-                Encrypt = true,
```

Comment:
Why did you remove this?

Reply:
Encrypt was removed because In memory database does not support it and I wanted to use in local memory db when working on new features. 
We could have this setting configurable if we want to have it on the azure but I am not sure if we want to encrypt data at all. I understand that encryption of the database data is useful when storing user sensitive data that needs to follow GDPR and while our app is not used with real data we can skip this. What do you think?

Reply:
Agreed.